docs/tutorials/common_data_operations.ipynb

```diff
@@ -0,0 +1,597 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Common Data Operations with `TAPE`"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this notebook, we'll highlight a handful of common dataframe operations that can be performed within `TAPE`. \n",
+    "\n",
+    "> **_Note:_**\n",
+    "TAPE extends the `Pandas`/`Dask` API, and so users familiar with those APIs can expect many operations to be near-identical when working with `TAPE`."
```

Comment:
Here we use TAPE and `TAPE` in the same line.

Do we want to always have it within ``?